test/functionality/dataflow/environments/modification-tests.ts

```diff
@@ -75,10 +67,8 @@ describe('Modification', () => {
 describe('Append', () => {
 	describe('Global', () => {
 		it('Different variables', () => {
-			let clean = initializeCleanEnvironments()
-			clean = define(variable('x', '_1'), GlobalScope, clean)
-			let append = initializeCleanEnvironments()
-			append = define(variable('y', '_2'), GlobalScope, append)
+			const clean = defaultEnvironment().defineEnv(variable('x', '_1', '_1', GlobalScope))
+			const append = defaultEnvironment().defineEnv(variable('y', '_2', '_2', GlobalScope))
```

Comment:
I am still not too sure about the `defineEnv(variable(` syntax. What is your opinion?